datasets/superb/README.md

```diff
@@ -301,7 +347,8 @@ The data fields in all splits are:
 
 #### er
 
-[More Information Needed](https://github.com/huggingface/datasets/blob/master/CONTRIBUTING.md#how-to-contribute-to-the-dataset-cards)
+- `file` (`string`): Path to the WAV audio file.
+- `label` (`string`): Label of the speech emotion.
```

Comment:
Same here: maybe we can have the mapping between the label ids (int) and the label names (str) here ?